tests/test-history.sh

```diff
@@ -11,11 +11,21 @@ USE_SYSTEMDIR=yes
 skip_without_bwrap
 skip_revokefs_without_fuse
 
-echo "1..1"
-
 HISTORY_START_TIME=$(date +"%Y-%m-%d %H:%M:%S")
 sleep 1
 
+if ! logger "Checking whether Flatpak can use the journal..."; then
+    skip "Cannot write to Journal with logger"
+fi
+
+messages="$(journalctl --user --since="${HISTORY_START_TIME}" || true)"
+
+if [ -z "$messages" ]; then
```

Comment:
So when you can read the log but there's nothing in it, it's a non-empty string?

Reply:
Ah, it's `-- No entries --` so I'll resolve my own question.

Reply:
The reason I logged a message with `logger` after `sleep 1` was precisely to make sure there would be something there.